packages/frontend-near-wallet/src/modules/redirects/pages/RedirectPage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { useLocation } from 'react-router-dom'
+import { getMyNearWalletUrl } from '../../../utils/constants';
+
+const RedirectPage = () => {
```

Comment:
Nice but I believe there is a screen here to prompt the user to select a wallet to redirect to cc @agileurbanite

Reply:
Not at the moment, except there's some design updates in this regard. We had a redirect prompt for when the transfer is initiated but @nall-near said we could remove it earlier.